resolwe/storage/connectors/googleconnector.py

```diff
@@ -52,13 +52,16 @@ def __getattr__(self, name):
     @validate_url
     def delete(self, url, urls):
         """Remove objects."""
-        with suppress(NotFound):
-            with self.client.batch():
-                for delete_url in urls:
-                    blob = self.bucket.blob(
-                        os.fspath(self.base_path / url / delete_url)
-                    )
-                    if blob.exists():
+        # At most 1000 objects can be deleted at the same time.
+        max_chunk_length = 1000
+        for i in range(0, len(urls), max_chunk_length):
+            with suppress(NotFound):
```

Comment:
Is it ok to put `suppress` here? Will be the rest of 1000 objects when one fails? Would it make more sense to put it only around `delete`?

Reply:
The documentation is not really clear but according to my experience all existing objects in the batch is deleted and the exception for last non-existing element in a batch is raised.